yaml/conf.go

```diff
@@ -0,0 +1,37 @@
+// Package yaml provides utility of reviewdog config yaml file.
+package yaml
```

Comment:
On second thought, I guess package should not be "yaml". "project" is better perhaps.